### ISSUE TITLE
Pods does not build (for me)

### ISSUE THREAD

**Issue by 1peter10** [dskleingeld/pods#41]
I want to test the app in order to add it to https://linmobapps.frama.io.
Sadly, compilation of current git master on Arch Linux ARM (cargo 1.52.0, rustc 1.52.1) with "cargo build --release"

~~~~~
Compiling pods v0.1.0 (/home/alarm/build/pods)
error[E0050]: method `update` has 2 parameters but the declaration in trait `iced::Application::update` has 3
  --> src/main.rs:85:15
   |
85 |     fn update(&mut self, message: Self::Message) -> Command<Self::Message> {
   |               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 3 parameters, found 2
   |
   = note: `update` from trait: `fn(&mut Self, <Self as Application>::Message, &mut iced::Clipboard) -> iced::Command<<Self as Application>::Message>`

error[E0063]: missing field `exit_on_close_request` in initializer of `iced::Settings<_>`
   --> src/main.rs:227:5
    |
227 |     Settings {
    |     ^^^^^^^^ missing `exit_on_close_request`

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0050, E0063.
For more information about an error, try `rustc --explain E0050`.
error: could not compile `pods`

To learn more, run the command again with --verbose.
~~~~~

If you need more info, please get in touch!

BTW: Are you guys aware of [this other project that tries to do something quite similar from what I can grasp](https://github.com/jnetterf/pyrocast)? The list of working features looks like it might be a great match.

**Comment by yara-blue** [comment]
Should be fixed now (f66a1f4) thanks for reporting!

Pyrocast looks great btw! When I started this I thought there was no good option on the Pinephone, it seems I was wrong! I think the LINMOB can really make a difference in app discovery. 

What I have now is really bare bones though it works for me, I am learning GUI programming on the go and decided to go for a pure rust approach to keep building simple (and cross platform).

**Comment by 1peter10** [comment]
Great, it works now!

Aside from GNOME Podcasts there wasn't much in November, sure, Pyroast was already around, but even I had not found it back then. It's always great to learn something, and while i would not call pods pretty, it's quite functiional! Thanks for your effort!
